pallets/attestation/src/lib.rs

```diff
@@ -35,13 +35,11 @@ mod tests;
 
 use sp_std::vec::Vec;
 
-pub use crate::attestations::*;
+pub use crate::{attestations::*, default_weights::WeightInfo};
 pub use pallet::*;
```

Comment:
I guess either merge everything or nothing?  😬 
```suggestion
pub use crate::{attestations::*, default_weights::WeightInfo, pallet::*};
```